python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -755,6 +866,15 @@ def process_edge_data(process_confs, node_id_map, num_processes):
 
     return edges, edge_data
 
+def verify_confs(confs):
+    """ Verify the configuration of the input data.
+    """
+    ntypes = {conf['node_type'] for conf in confs["node"]}
+    etypes = [conf['relation'] for conf in confs["edge"]]
+    for src_type, _, dst_type in etypes:
```

Comment:
What if a node type does not have any node feature? Are we still ask users to provide node files for that node type?

Reply:
yes, we currently require users to provide node files for every node type. computing unique node IDs is very expensive operation. it requires appropriate optimization. I don't think I'll support this in v0.1. I want to see users' feedbacks first.